app/ApiObjects/User.php

```diff
@@ -0,0 +1,30 @@
+<?php namespace App\ApiObjects;
+
+use App\ApiObjects\User as ApiUser;
+use Illuminate\Contracts\Support\Arrayable;
+use Illuminate\Contracts\Support\Jsonable;
+use Illuminate\Support\Facades\Auth;
+
+class User implements Arrayable, Jsonable
+{
+    protected $user;
+
+    public function __construct($user)
+    {
+        $this->user = $user;
+    }
+
+    public function toJson($options = 0)
```

Comment:
Replied to other comment:

I know why you did that! The method has to be compatible with that of its parent class `Illuminate\Contracts\Support\Jsonable::toJson()`. Spits out errors if that `$options` parameter isn't there.